src/main/resources/staff/templates/document/sealedClaim.html

```diff
@@ -88,25 +88,24 @@
           alt=""/>
       </td>
-      <td colspan="2"><p>In the County Court Business Centre Online Court</p></td>
+      <td><p>In the County Court Business Centre Online Court</p></td>
     </tr>
     <tr>
-      <td><h2><strong>Claim number: </strong></h2></td>
-      <td><h2 style="font-weight: normal"> {{ claim.referenceNumber }}</h2></td>
+      <td><h2 style="font-weight: normal"><strong>Claim number:</strong> {{ claim.referenceNumber }}</h2></td>
     </tr>
     <tr>
-      <td colspan="2">
+      <td>
         <p>{{ claimant.fullName }}’s claim</p>
       </td>
     </tr>
     <tr>
-      <td colspan="2"><p><strong>Submitted on: </strong> {{ claim.submittedOn }}</p></td>
+      <td><p><strong>Submitted on: </strong> {{ claim.submittedOn }}</p></td>
     </tr>
     <tr>
-      <td colspan="2"><p><strong>Issued on: </strong> {{ claim.issuedOn }}</p></td>
+      <td><p><strong>Issued on: </strong> {{ claim.issuedOn }}</p></td>
```

Comment:
beware long months, this was changed to 2 months for September I think. Could you test with September and make sure there's no wrapping please?

Reply:
@timja-kainos Why would there be any wrapping introduced here? Removing `colspan` from these cells resulted in there being one less column overall, so the cells that had `colspan="2"` should be as wide now as they were then. Well, at least unless there are some column sizing styles messing it up, but it doesn't seem like it.

Reply:
sounds good, was just a little worried, because we had to make changes to make September fit, we first built them in July and may not have tested longer months